pack/Dockerfile

```diff
@@ -601,6 +601,9 @@ ENV NVIDIA_DISABLE_REQUIRE="true" \
     NVIDIA_VISIBLE_DEVICES="all" \
     NVIDIA_DRIVER_CAPABILITIES="compute,utility"
 
+# Force GPUStack auth for localhost request as higress will redirect request to localhost
+ENV GPUSTACK_FORCE_AUTH_LOCALHOST='true'
```

Comment:
Is a false value represents disabling auth now?

Reply:
`false(default)` means disabling auth for localhost now.